project.clj

```diff
@@ -28,14 +29,13 @@
                  [org.clojure/clojurescript "0.0-3308"]
                  [org.omcljs/om "0.8.8"]
                  [secretary "1.2.3"]
-                 [prismatic/om-tools "0.3.11"]
                  [com.andrewmcveigh/cljs-time "0.3.0"]
                  
                  [environ "1.0.0"]
 
                  [org.clojure/tools.cli "0.3.3"]
                  [org.clojars.hozumi/clj-commons-exec "1.0.6" ]]
-  :description "ACeDB emulator and migration tools"
+  :description "REST API for getting data from datomic on a per widget basis"
   :source-paths ["src/clj/pseudoace", "src/clj/", "src/perl/"]
```

Comment:
You can probably remove "src/clj/pseudoace" and "src/prl" from this vector.
Other than that, this is good to merge.